tests/test_angleunit.py

```diff
@@ -66,7 +66,7 @@ def test_invalid():
     # Wrong type of value argument
     np.testing.assert_raises(TypeError, coord.AngleUnit, coord.degrees)
     # Also wrong type, but strings give a ValueError
-    np.testing.assert_raises(ValueError, coord.AngleUnit, 'spam')
+    np.testing.assert_raises((ValueError, TypeError), coord.AngleUnit, 'spam', coord.degrees)
```

Comment:
```suggestion
    # the jax-galsim version of this class raises a TypeError here
    # instead of a ValueError
    np.testing.assert_raises((ValueError, TypeError), coord.AngleUnit, 'spam', coord.degrees)
```